src/backend.ts

```diff
@@ -122,13 +122,16 @@ export class Backend<M extends IAnyModelType> {
     return false;
   }
 
-  async realProcessAll() {
+  async realProcessAll(liveOnly?: boolean) {
```

Comment:
Don't think this parameter is optional, since it will always receive one through the `processAll` method. Or does typescript complain about a possibly null argument from the state.ts file (`processAll` method).

Reply:
It has to be nullable as the parameter passed in state.ts is optional